test_e2e/python/test-ingest-nomicdb.py

```diff
@@ -0,0 +1,80 @@
+import os
```

Comment:
In general, we've moved to adding integration tests rather than e2e tests for new connectors. Take a look at this s3 example: [test_s3.py](https://github.com/Unstructured-IO/unstructured-ingest/blob/main/test/integration/connectors/test_s3.py). This should help isolate chunk the connector code and make testing it easier.